egograph/backend/infrastructure/database/youtube_queries.py

```
@@ -115,204 +81,174 @@ def _generate_partition_paths(
-        ORDER BY w.watched_at_utc DESC
+            watch_event_id,
+            watched_at_utc,
+            video_id,
+            video_url,
+            video_title,
+            channel_id,
+            channel_name,
+            content_type
+        FROM enriched_watch_events
+        ORDER BY watched_at_utc DESC
     """
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)